doc/generic/pgf/text-en/pgfmanual-en-base-quick.tex

```diff
@@ -102,7 +102,7 @@ \subsection{Quick Path Construction Commands}
     Adds a radius around the origin of the given \meta{radius}. This command is
     orders of magnitude faster than |\pgfcircle{\pgfpointorigin}{|\meta{radius}|}|.
     %
-\begin{codeexample}[]
+\begin{codeexample}[pre={\colorlet{examplefill}{yellow!80!black}}]
```

Comment:
I think `\colorlet{examplefill}{yellow!80!black}` can even go inside the `codeexample` like so
```suggestion
\begin{codeexample}[]
\colorlet{examplefill}{yellow!80!black}
```